falcon/media/multipart.py

```diff
@@ -53,8 +53,8 @@ class MultipartParseError(errors.HTTPBadRequest):
     """
 
     def __init__(self, description=None, headers=None, **kwargs):
-        super().__init__('Malformed multipart/form-data request media',
-                         description, headers, **kwargs)
+        super().__init__(title='Malformed multipart/form-data request media',
```

Comment:
maybe this could also use the decorator `deprecated_args`?

Reply:
@CaselIT yeah, I'll clean it up in a follow-up PR.